tests/failpoints/cases/test_import_service.rs

```diff
@@ -279,45 +279,84 @@ fn test_ingest_sst_v2() {
+    fail::cfg("on_flush_completed", "return()").unwrap();
     let resp = import.ingest(&ingest).unwrap();
     assert!(!resp.has_error(), "{:?}", resp.get_error());
-    fail::cfg("on_cleanup_import_sst", "return").unwrap();
     let (tx, rx) = channel::<()>();
     let tx = Arc::new(Mutex::new(tx));
     fail::cfg_callback("on_cleanup_import_sst_schedule", move || {
         tx.lock().unwrap().send(()).unwrap();
     })
     .unwrap();
     rx.recv_timeout(std::time::Duration::from_secs(20)).unwrap();
-    let mut count = 0;
-    for path in &cluster.paths {
-        let sst_dir = path.path().join("import-sst");
-        for entry in std::fs::read_dir(sst_dir).unwrap() {
-            let entry = entry.unwrap();
-            if entry.file_type().unwrap().is_file() {
-                count += 1;
-            }
-        }
-    }
+    let count = sst_file_count(&cluster.paths);
+    assert_eq!(1, count);
+    fail::remove("on_cleanup_import_sst_schedule");
 
+    // region size and keys should be updated.
     let (tx, rx) = channel::<()>();
     let tx = Arc::new(Mutex::new(tx));
     fail::cfg_callback("on_update_region_keys", move || {
         tx.lock().unwrap().send(()).unwrap();
     })
     .unwrap();
     rx.recv_timeout(std::time::Duration::from_secs(20)).unwrap();
-
     fail::remove("on_update_region_keys");
-    fail::remove("on_cleanup_import_sst");
-    fail::remove("on_cleanup_import_sst_schedule");
-    assert_ne!(0, count);
-
-    std::thread::sleep(std::time::Duration::from_secs(1));
+    std::thread::sleep(std::time::Duration::from_millis(100));
 
     let region_keys = cluster
         .pd_client
         .get_region_approximate_keys(ctx.get_region_id())
         .unwrap();
     assert_eq!(100, region_keys);
+
+    // restart node
+    cluster.stop_node(1);
+    cluster.start().unwrap();
+    let count = sst_file_count(&cluster.paths);
+    assert_eq!(1, count);
+
+    // flush manual
+    fail::remove("on_flush_completed");
+    let (tx, rx) = channel::<()>();
+    let tx = Arc::new(Mutex::new(tx));
+    fail::cfg_callback("on_flush_completed", move || {
+        tx.lock().unwrap().send(()).unwrap();
+    })
+    .unwrap();
+    for i in 0..count {
+        cluster.must_put(format!("k-{}", i).as_bytes(), b"v");
+    }
+    cluster.flush_data();
+
+    rx.recv_timeout(std::time::Duration::from_secs(20)).unwrap();
+    fail::remove("on_flush_completed");
+    std::thread::sleep(std::time::Duration::from_millis(100));
+    let count = sst_file_count(&cluster.paths);
+    assert_eq!(0, count);
+}
+
+fn sst_file_count(paths: &Vec<TempDir>) -> u64 {
+    let mut count = 0;
+    for path in paths {
+        let sst_dir = path.path().join("import-sst");
+        for entry in std::fs::read_dir(sst_dir).unwrap() {
+            let entry = entry.unwrap();
+
+            if entry
+                .path()
+                .file_name()
+                .and_then(|n| n.to_str())
+                .unwrap()
+                .contains("0_0_0")
```

Comment:
what is this?

Reply:
In this begin, import wiil  create one file named as `0_0_0_0.sst`